arm-storageimportexport/2016-11-01/swagger/storageimportexport.json

```diff
@@ -0,0 +1,1209 @@
+    "version": "2016-11-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.windows-ppe.net/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.ImportExport/locations": {
+      "get": {
+        "tags": [
+          "StorageImportExport"
+        ],
+        "operationId": "ListLocations",
+        "description": "Returns a list of locations to which you can ship the disks associated with an import or export job. A location is a Microsoft data center region.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/Accept-Language"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/LocationsListResult"
+            }
+          },
+          "default": {
+            "description": "An error occurs.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.ImportExport/locations/{locationName}": {
+      "get": {
+        "tags": [
+          "StorageImportExport"
+        ],
+        "operationId": "GetLocation",
+        "description": "Gets a location to which you can ship the disks associated with an import or export job. A location is an Azure region.",
+        "parameters": [
+          {
+            "name": "locationName",
+            "in": "path",
+            "description": "The name of the location. For example, “West US” or “westus”.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/Accept-Language"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/Location"
+            }
+          },
+          "default": {
+            "description": "An error occurs.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ImportExport/jobs": {
+      "get": {
+        "tags": [
+          "StorageImportExport"
+        ],
+        "operationId": "Jobs_ListBySubscription",
```

Comment:
Operations ids we had before review:
"operationId": "ListLocations",
"operationId": "GetLocation",
"operationId": "ListJobsInSubscription",
"operationId": "ListJobsInResourceGroup",
"operationId": "GetJob",
"operationId": "UpdateJob",
"operationId": "PutJob",
"operationId": "DeleteJob",
"operationId": "Movejobs",
"operationId": "GetBitLockerKeys",
"operationId": "ListAvailableOperations",

After review:
"operationId": "ListLocations",
"operationId": "GetLocation",
"operationId": "Jobs_ListBySubscription",
"operationId": "Jobs_ListByResourceGroup",
"operationId": "Jobs_Get",
"operationId": "Jobs_Update",
"operationId": "Jobs_CreateOrUpdate",
"operationId": "Jobs_Delete",
"operationId": "Jobs_Move",
"operationId": "Jobs_ListBitLockerKeys",
"operationId": "ListSupportedOperations",

1) The change doesn't look consistent.
2) List Jobs can be per Resource Group vs. per Subscription, hence the need to distinguish them.